paasta_tools/oom_logger.py

```diff
@@ -136,11 +162,17 @@ def capture_oom_events_from_stdin():
                 break
 
 
-def get_container_env_as_dict(docker_inspect):
+def get_container_env_as_dict(
+    is_cri_containerd: bool, container_inspect: dict
+) -> Dict[str, str]:
     env_vars = {}
-    config = docker_inspect.get("Config")
-    if config is not None:
+    if is_cri_containerd:
+        config = container_inspect.get("process")
+        env = config.get("env", [])
+    else:
+        config = container_inspect.get("Config")
         env = config.get("Env", [])
+    if config is not None:
```

Comment:
Technically this `if config is not None` should never be hit if we've got that far; if `config` was none we'd run into an exception when reaching into it while trying to get the `env` in each of the cases above.
Maybe this should be more like: 

```suggestion
    if is_cri_containerd:
        config = container_inspect.get("process")
        env_key = "env"
    else:
        config = container_inspect.get("Config")
        env_key = "Env"
    if config is not None:
        env = config.get(env_key, [])
        ...
```